components/elm/src/dyn_subgrid/dynSubgridControlMod.F90

```diff
@@ -231,12 +231,8 @@ subroutine check_namelist_consistency
     end if
 
     if (dyn_subgrid_control_inst%do_harvest) then
-       if (.not. use_cn) then
-          write(iulog,*) 'ERROR: do_harvest can only be true if use_cn is true'
-          call endrun(msg=errMsg(sourcefile, __LINE__))
-       end if
-       if (use_fates) then
-          write(iulog,*) 'ERROR: do_harvest currently does not work with use_fates'
+       if (.not. (use_cn .or. use_fates)) then
```

Comment:
@aldivi Here is the code that ensures harvest is turned on only when BGC is also turned on. Thus, your modifications to the harvest code should not change SP mode. Do you agree?